<?xml version="1.0"?>
<launch>
  <!-- implement this to make things easier for later -->
  <arg name="params_file" default="ur5e_workstation_sim_params" />

  <include file="$(find uml_hri_nerve_pick_and_place)/launch/gpd_ur5.launch" />

  <include file="$(find uml_hri_nerve_armada_workstation)/universal_robot/ur_e_gazebo/launch/gazebo_universal_robot.launch" >
    <arg name="model" value="ur5"/>
    <arg name="sim_workstation" value="true"/>
    <arg name="gripper" value="robotiq_2f_85"/>
    <arg name="wrist_cam" value="d435_millibar"/>  <!-- d435_millibar -->
  </include>

  <rosparam file="$(find uml_hri_nerve_pick_and_place)/cfg/ur5e_wrist_d435.yaml" />

  <node name="pick_and_place_gazebo" pkg="uml_hri_nerve_pick_and_place" type="pick_and_place_gazebo" output="screen"/>
  <node name="spawn_object" pkg="uml_hri_nerve_pick_and_place" type="spawn_object.py" output="screen"/>

</launch>
